src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
     permissionObj['resource']['resourceIdentifier'] = folderId
   }
 
-  const links: { url: string; label: string }[] = [
-    {
-      url: routes.toCustomDashboardHome({ accountId }),
-      label: 'Home'
-    },
-    {
-      url:
-        folderId === 'shared' ? routes.toCustomDashboardHome({ accountId }) : routes.toCustomFolderHome({ accountId }),
-      label: folderId === 'shared' ? getString('common.dashboards') : getString('dashboards.homePage.folders')
-    }
-  ]
-
-  if (folderId) {
-    links.push({
-      url: routes.toCustomDashboardHome({ accountId }),
-      label: title
-    })
-  }
+  React.useEffect(() => {
+    const links = getBreadcrumbLinks(folderDetail, accountId, getString('dashboards.homePage.folders'))
+    includeBreadcrumbs(links)
```

Comment:
Might as well eliminate the variable.
```suggestion
    includeBreadcrumbs(
      getBreadcrumbLinks(folderDetail, accountId, getString('dashboards.homePage.folders'))
    )
```